src/main/scala/mesosphere/marathon/core/deployment/impl/DeploymentActor.scala

```diff
@@ -247,9 +246,9 @@ object DeploymentActor {
     Props(new DeploymentActor(
       deploymentManagerActor,
       killService,
+      schedulerActions,
       scheduler,
```

Comment:
I'm not sure we should keep the `schedulerActions` around.

Reply:
It has been a thorn in my side for a while now. However, it's logic should be a part of the _Orchestrator_ and not to the _Scheduler_. Nevertheless, we should make it use the new _Scheulder_ instead of _LaunchQueue_ and _KillService_. Wdyt?

Reply:
We can move it in this diff but then we have to introduce the `scheduler.create` or something that uses the launch queue. It is not trivial.

Reply:
I'm not saying we should remove _LaunchQueue_ completely in this phase, but at least Deployment* actors could switch to using the new scheduler. Otherwise, this looks cumbersome and inconsistent. I'd be glad to help with this if you want to pair.